CHANGELOG.next.toml

```diff
@@ -72,3 +72,9 @@ wired up by default if none is provided.
 references = ["smithy-rs#1603", "aws-sdk-rust#586"]
 meta = { "breaking" = true, "tada" = false, "bug" = false, "target" = "client" }
 author = "jdisanti"
+
+[[smithy-rs]]
+message = "Flatten and remove Smithy IDLv2 model mixins to not influence generated code output"
```

Comment:
Shouldn't the changelog entry be geared for users? I would have expected this to read something along the lines of "Smithy IDL v2 mixins are now supported", linking to the relevant documentation.

Reply:
Updated.